src/main/java/Duke.java

```diff
@@ -1,10 +1,95 @@
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
+    public static void main(String[] args) throws DukeException {
+        String logo = "Hello! I'm Duke\n" + "What can I do for you?\n";
+        System.out.println(logo);
+        Scanner in = new Scanner(System.in);
+        String response;
+
+        List<Task> list = new ArrayList<>();
+        while (true) {
+            response = in.nextLine();
+            if (response.equals("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+            try {
+                if (response.equals("list")) {
+                    int size = list.size();
+                    if (size == 0) {
+                        System.out.println("Bro.... You have nothing on your list. Add something to your list to be productive");
+                        continue;
+                    }
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < size; i++) {
+                        System.out.println((i + 1) + "." + list.get(i).toString());
+                    }
```

Comment:
Please separate them into several methods instead of putting everything in the `main()`. Such as, `displayAllTasks()`, `AddNewTask()`....